proxy/http2/Http2ConnectionState.cc

```diff
@@ -1174,9 +1174,9 @@ Http2ConnectionState::send_a_data_frame(Http2Stream *stream, size_t &payload_len
 void
 Http2ConnectionState::send_data_frames(Http2Stream *stream)
 {
-  if (stream->get_state() == HTTP2_STREAM_STATE_CLOSED || stream->get_state() == HTTP2_STREAM_STATE_HALF_CLOSED_LOCAL) {
+/*  if (stream->get_state() == HTTP2_STREAM_STATE_CLOSED || stream->get_state() == HTTP2_STREAM_STATE_HALF_CLOSED_LOCAL) {
     return;
-  }
+  } */
```

Comment:
Ok, I'll try adjusting the fix to just call the delete stream cleanup if this is called when already closed.